### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_MapDataSrcObjects_Keyboard: On expanding tree view items with the help enter key ,keyboard focus directly shift to the fifth "Summary" screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6153]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS3.3.2; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS :Windows_NT x64 10.0.18362

**Additional Details:**
Tools Name: Keyboard
MAS References: MAS3.2.2

**Steps to Reproduce:**
1. Install the app Azure Data Studio.
2. Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button.
3. Select any tree view item from left panel using arrow key and press enter.
4. Observe that keyboard focus directly shift to fifth "Summary" screen. 

**Actual:**
On expanding tree view items with the help enter key ,keyboard focus directly shift to the fifth "Summary" screen. 

**Expected:**
On expanding tree view items with the help enter key ,keyboard focus remains on the same screen .

**User Impact:**
Keyboard user will get confuse with the unauthorize behaviour of the expand tree view, because when user expand any tree item and enter he will redirect to the next page and content on the first page will miss out. 

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference :
MAS3.2.2:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={24cbfff7-374e-42e6-b9ab-d87fcd53b3e8}

**Attachment for Reference:**
[6153_[Accessibility]A11y_AzureDataStudio_MapDataSrcObjects_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3324797/6153_.Accessibility.A11y_AzureDataStudio_MapDataSrcObjects_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
@AccessibilityTestingTeam-TCS - I wanted to confirm, but is tree expansion toggle on enter press a standard we should be following everywhere? The base tree control we're using here doesn't have that by default (it just selects the item) so if that is the intended behavior it seems like we should open a bug against VS Code to fix that there.

**Comment by Charles-Gagnon** [comment]
@AccessibilityTestingTeam-TCS The current behavior is to toggle the expansion state upon pressing enter on the node (so it'll switch to opened/closed based on the current state of the node). Let me know if this isn't the desired behavior - for example if it should be just expanding the node on enter and not collapsing it if it's already opened.
